sources/packages/web/src/services/OverawardService.ts

```diff
@@ -45,8 +45,8 @@ export class OverawardService {
    * @returns primary identifier of the resource created.
    */
   async addManualOverawardDeduction(
-    studentId: number,
     payload: OverawardManualRecordAPIInDTO,
+    studentId?: number,
```

Comment:
Why did we go with this approach.?

Reply:
Reverted it back, was doing it for a different change.

Reply:
@dheepak-aot @ann-aot I believe that it happened by a mistake/misunderstood in one of the conversations that I had with @guru-aot. I was trying to get rid of the `props.studentId as number` but forgot that the method would be not used by the student.
Either way, I would still prefer a change like below to avoid the `as number` in that context that looks more like a workaround.
![image](https://user-images.githubusercontent.com/61259237/228638829-9c372907-2626-4256-89a8-89c41ae280fb.png)